db/migrate/20240624121212_katello_recreate_evr_constructs.rb

```diff
@@ -0,0 +1,159 @@
+class KatelloRecreateEvrConstructs < ActiveRecord::Migration[6.1]
+  def change
+    count = select_value <<~SQL
+      SELECT count(*) FROM pg_extension WHERE extname = 'evr';
+    SQL
+    if count.to_i == 0
+      return
+    else
+      execute <<~SQL
+        DROP EXTENSION evr CASCADE;
+      SQL
```

Comment:
There's a native DSL for this:
```suggestion
     disable_extension('evr', force: true)
```

That also properly reloads the type map.

Reply:
I will create a follow up PR with these updates..The earlier migration was indeed using enable_extension. 

Do you know if the DSL surpasses the privilege restrictions somehow?

Reply:
No, it's using the same SQL